docs/saveWater.md

```diff
@@ -1,3 +1,21 @@
 # Save Water
 
-> Oops! Seems that we don't have documentation for this method yet. Feel free to contribute opening a new PR
+Add a specified amount of consumed water to Healthkit
+
+`saveWater` accepts an options object containing a numeric water value in liters. 
+In the example below 200 ml of water are added to Healthkit.
+```javascript
+let options = {
+  water: 0.2 // Amount in liters
```

Comment:
We can add the `date` key into the options too, so it shows developers that we can set the time when this sample was consumed

Would be something like

```javascript
let options = {
  water: 0.2, // Amount in liters
  date: (new Date(2018,10,1)).toISOString(), // Time when the water amount was consumed
}
```

What do you think?

Reply:
Sure, go ahead! Maybe add that the date is optional and will default to the moment the amount was added (I figure this will suffice for most use cases).